### PR TITLE
PR to fix Issue #877: terraform plan shows error in AWS

### PR DESCRIPTION
**Pull request by yudaykiran**
**What this PR does / why we need it**:
- Updated code to download the latest version of terraform.

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #
Fix for Issue #877.

**Special notes for your reviewer**:

Signed-off-by: yudaykiran <udaykiran.y@gmail.com>

### CODE REVIEW

File: e2e/terraform/oebs-cloud.sh [hunk 1]
```diff
@@ -99,8 +100,9 @@ function setup_local_env() {
         sleep 2
     else 
         echo "Missing terraform; Downloading and installing..."
-        wget "https://releases.hashicorp.com/terraform/0.10.0/terraform_0.10.0_linux_amd64.zip"
-        unzip terraform_0.10.0_linux_amd64.zip
+        terraform_url=$(curl https://releases.hashicorp.com/index.json | jq '{terraform}' | egrep "linux.*amd64" | sort --version-sort -r | egrep -v 'beta|rc' | head -1 | awk -F[\"] '{print $4}')
```
Comment (ksatchit) [inline e2e/terraform/oebs-cloud.sh]: Good one ! Any thoughts about egrep v/s grep -e , most linux flavours do support it though.

Comment (yudaykiran) [reply e2e/terraform/oebs-cloud.sh]: egrep is equivalent to using "grep -E". Do you suggest I use "grep -E" instead?

Comment (ksatchit) [reply e2e/terraform/oebs-cloud.sh]: Right. grep -e would be safer, I suppose.

Comment (yudaykiran) [reply e2e/terraform/oebs-cloud.sh]: Done, changed "egrep" to "grep -E".

**Pull request merged**